src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  padding: 1px;
+}
+
+.stars--1:nth-child(1) {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2:nth-child(2) {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3:nth-child(3) {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4:nth-child(4) {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5:nth-child(5) {
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
There is a lot of repeated code here. To avoid this, you can use a CSS preprocessor like SASS or LESS, which supports variables and mixins. This will help you to keep your code DRY (Don't Repeat Yourself).